addons/mail/static/src/components/attachment/attachment_tests.js

```diff
@@ -595,6 +595,47 @@ QUnit.test('close attachment viewer', async function (assert) {
     );
 });
 
+QUnit.test('delete an attachment', async function (assert) {
+    assert.expect(2);
+
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === "unlink" && args.model === "ir.attachment") {
+                assert.step('attachment_unlink');
+                return;
+            }
+            return this._super(...arguments);
+        },
+    });
+    const attachment = this.env.models['mail.attachment'].create({
+        filename: "test.txt",
+        id: 750,
+        mimetype: 'text/plain',
+        name: "test.txt",
+    });
+    await this.createAttachmentComponent(attachment, {
+        detailsMode: 'hover',
+        isDownloadable: false,
+        isEditable: true,
+        showExtension: false,
+        showFilename: true,
+    });
+    await afterNextRender(() => {
+        document.querySelector('.o_Attachment_actionUnlink').click();
+    });
+
+    await afterNextRender(() => {
+        // ensures that the unlink method is called only once, regardless of the amount of times the button is pressed.
```

Comment:
this comment is good but superfluous, it should be the test title + the final assert comment instead